src/app/pages/home/components/onboarding-step-item.tsx

```diff
@@ -16,24 +18,31 @@ interface StepIllustrationProps {
 }
 const StepIllustration = ({ image }: StepIllustrationProps) => <img src={image} />;
 
-const PopupDoneIcon = () => (
-  <Circle border="1px solid" borderColor={color('border')} color={color('bg')} size="40px">
-    <FiCheck color={color('accent')} />
-  </Circle>
-);
-
 interface OnboardingStepItemProps {
   action: string;
   body: string;
   imageFull: string;
+  imageFullDone: string;
   imagePopup: string;
+  imagePopupDone: string;
   isDone: boolean;
   onClick(): void;
   routeType: number;
   title: string;
 }
 export function OnboardingStepItem(props: OnboardingStepItemProps) {
-  const { action, body, imageFull, imagePopup, isDone, onClick, routeType, title } = props;
+  const {
+    action,
+    body,
+    imageFull,
+    imageFullDone,
+    imagePopup,
+    imagePopupDone,
+    isDone,
+    onClick,
+    routeType,
```

Comment:
I know this hasn't changed, but just thinking this isn't necessary? Can't we infer whether a link is internal or external? `isExternal... startsWith('https://')`?

Reply:
Yep, I can do it this way.